projects/uitdatabank/reference/entry.json

```diff
@@ -9586,6 +9586,184 @@
         ]
       }
     },
+    "/users/{userId}": {
+      "get": {
+        "summary": "users - userid - get",
+        "operationId": "users-userId-get",
+        "description": "Returns the details of the user with the corresponding userId.",
+        "tags": [
+          "Users"
+        ],
+        "responses": {
+          "200": {
+            "description": "The user details.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "uuid": {
+                      "type": "string",
+                      "example": "c8e6ff82-8b30-4295-b937-ab2f4f6ab4bf",
+                      "format": "uuid",
+                      "description": "UUID of the user."
+                    },
+                    "email": {
+                      "type": "string",
+                      "example": "info@publiq.be",
+                      "description": "The email address of the user."
+                    },
+                    "username": {
+                      "type": "string",
+                      "example": "username",
+                      "description": "The username of the user."
+                    },
+                    "id": {
```

Comment:
I don't think we expose `id` and `nick` on the calls to get a user by ID or email. It is indeed exposed when getting the current user because it is added here https://github.com/cultuurnet/udb3-backend/blob/master/src/Http/User/GetCurrentUserRequestHandler.php#L43-L45

But I don't see that inside `GetUserByIdRequestHandler` or `GetUserByEmailRequestHandler`